AmazonChimeSDK/AmazonChimeSDKTests/tools/VideoFrameGenerator.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  VideoFrameGenerator.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import UIKit
+import AmazonChimeSDK
+
+class VideoFrameGenerator {
```

Comment:
nit: Is this in the global namespace? Maybe would be nice to prefix it with 'Test', i.e. 'TestVideoFrameGenerator'